test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -0,0 +1,417 @@
+---------------------------------------------------------------------------------------------------
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+--[[ Required Shared libraries ]]
+local commonRC = require("test_scripts/RC/commonRC")
+local test = require("user_modules/dummy_connecttest")
+
+--[[ Local Variables ]]
+local c = {}
+c.modules = { "RADIO", "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
+
+function c.preconditions()
+  commonRC.preconditions()
+end
+
+function c.start()
+  commonRC.start(nil, test)
+end
+
+function c.AddOnRCStatusToPT(tbl)
+  tbl.policy_table.functional_groupings.RemoteControl.rpcs.OnRCStatus = {
+    hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" }
+  }
+end
+
+function c.getRCAppConfig()
+  local struct = commonRC.getRCAppConfig()
+  struct.moduleType = c.modules
+  return struct
+end
+
+local function PTUfunc(tbl)
+  c.AddOnRCStatusToPT(tbl)
+  local appId = config.application1.registerAppInterfaceParams.appID
+  tbl.policy_table.app_policies[appId] = c.getRCAppConfig()
+end
+
+function c.raiPTUn(pPTUfunc, pAppId)
+  if not pAppId then pAppId = 1 end
+  if not pPTUfunc then
+    pPTUfunc = PTUfunc
+  end
+  commonRC.rai_ptu_n(pAppId, pPTUfunc, test)
+end
+
+function c.raiN(pAppId)
+  commonRC.rai_n(pAppId, test)
+end
+
+function c.getHMIAppId(pAppId)
+  local appId = commonRC.getHMIAppId(pAppId)
+  return appId
+end
+
+function c.getModuleControlData(pModuleType)
+  local struct = {}
+  if "CLIMATE" == pModuleType then
+    struct.moduleType = "CLIMATE"
+    struct.climateControlData = {}
+    struct.climateControlData.heatedSteeringWheelEnable = true
+    struct.climateControlData.heatedWindshieldEnable = true
+    struct.climateControlData.heatedRearWindowEnable = true
+    struct.climateControlData.heatedMirrorsEnable = true
+  elseif "RADIO" == pModuleType then
+    struct.moduleType = "RADIO"
+    struct.RadioControlData = {}
+    struct.RadioControlData.sisData = {
+      stationShortName = "Name1",
+      stationIDNumber = {
+        countryCode = 100,
+        fccFacilityId = 100
+      },
+      stationLongName = "RadioStationLongName",
+      stationLocation = {
+        longitudeDegrees = 0.1,
+        latitudeDegrees = 0.1,
+        altitudeMeters = 0.1
+      },
+      stationMessage = "station message"
+    }
+  elseif "AUDIO" == pModuleType then
+    struct.moduleType = "AUDIO"
+    struct.audioControlData = {
+      source = "RADIO_TUNER",
+      keepContext = false,
+      volume = 50,
+      equalizerSettings = {
+        channelId = 10,
+        channelName = "Channel 1",
+        channelSetting = 50 }
+    }
+  elseif "LIGHT" == pModuleType then
+    struct.moduleType = "LIGHT"
+    struct.lightControlData = {
+      lightState = {
+        id = "FRONT_LEFT_HIGH_BEAM",
+        status = "ON",
+        density = 0.2,
+        sRGBColor = "red"
```

Comment:
@dboltovskyi @GetmanetsIrina sRGBColor is a structure, so there should be something like 
```
        sRGBColor = {
        red = 10,
        green = 10,
        blue = 10
        }
```